samples/cpp/visual_language_chat/load_image.cpp

```diff
@@ -0,0 +1,41 @@
+
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#define STB_IMAGE_IMPLEMENTATION
+#include "stb_image.h"
+#include "load_image.hpp"
+
+ov::Tensor utils::load_image(const std::filesystem::path& image_path) {
+    int x = 0, y = 0, channels_in_file = 0;
+    constexpr int desired_channels = 3;
+    unsigned char* image = stbi_load(
+        image_path.string().c_str(),
+        &x, &y, &channels_in_file, desired_channels);
+    if (!image) {
+        throw std::runtime_error{"Failed to load the image"};
```

Comment:
not meaningful: we can use OPENVINO_ASSERT instead of custom exceptions

Reply:
That's a sample. I don't think people should use OV exceptions. They should have their own type